### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Welcome_Resources_ScreenReader: The action performed of copying is not announced by screen reader.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#3156]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:-A11y_AzureDataStudioStayHealthy;-A11ySev2;-A11yTCS;-A11yWCAG2.1;-NonBenchmark-DesktopApp;-MAS4.1.3;-SQL Azure Data Studio;-FTP;-Keyboard;-STP;-NVDA;-JAWS;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS4.1.3
Screen reader: NVDA2020.1, JAWS

**Repro Steps:**
1 . Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 . Navigate till Resources section on Welcome screen
4 .Launch screen reader as NVDA
5 .Navigate to documentation link hit enter and tab to copy button and press enter.

**Actual:**
On activating the copy button in the dialog the action performed is not announced by the screen reader.

**Issue observed in the following screens:**
**Videos Pop Up**

**Expected:**
When the copy button is activated screen reader should announce data has been copied and a visual cue as well should be provided.

**User Impact:**
Keyboard and screen reader users will not be aware of the important change that has occurred 

**Attachment link for Reference:**
